client/src/index.jsx

```diff
@@ -29,20 +32,41 @@ class Header extends React.Component {
     let endpoint = '/funding/' + splitComponentUrl[3];
     axios.get(urlWithoutEndpoint + endpoint)
       .then((fundingData) => {
+        console.log('fundingData', fundingData);
         this.setState({
           'fundingGoal': fundingData.data.backing.fundingGoal,
           'amountFunded': fundingData.data.backing.amountFunded,
           'backers': fundingData.data.backing.backers,
-          // 'description': fundingData.data.backing.description,
+          'description': fundingData.data.backing.description,
+          'title': fundingData.data.backing.title,
           'backers': fundingData.data.backing.backers,
           'daysRemaining': fundingData.data.backing.daysRemaining,
         });
+        this.changeProgress();
       })
       .catch((err) => {
         throw new Error(err);
       });
   }
 
+  changeProgress() {
+    let element = document.getElementById("progressBar");
+    console.log('element', element);
+    let percent = this.state.fundingGoal / this.state.amountFunded;
+    console.log('percent', percent);
+    if (percent < 1) {
+      console.log('element.style.width', element.style.width);
+      element.style.width = percent * 100 + "%";
+      console.log('element.style.width', element.style.width);
+    } else {
+      element.style.width = 100 + '%';
+    }
+  }
+
+  componentDidMount() {
```

Comment:
Is this function call for changeProgress needed here? When you're already calling it on line 45?